addons/web/static/src/views/fields/monetary/monetary_field.js

```diff
@@ -64,20 +64,28 @@ export class MonetaryField extends Component {
         return getCurrency(this.currencyId).digits;
     }
 
+    get recordValue(){
```

Comment:
sounds reasonable to me

Reply:
me too, but I think we typically call it `value`. Also, don't forget to run the prettier script please